python/vyos/ipsec.py

```diff
@@ -83,6 +83,32 @@ def get_vici_sas_by_name(ike_name: str, tunnel: str) -> list:
         raise ViciCommandError(f'Failed to get SAs')
 
 
+def get_vici_connection_by_name(ike_name: str) -> list:
+    """
+    Find loaded SAs by IKE_SA name and return list with SASs info
+    :param ike_name: IKE SA name
+    :type ike_name: str
+    :param tunnel: CHILD SA name
+    :type tunnel: str
+    :return: list of Ordinary Dicts with SASs
+    :rtype: list
+    """
+    from vici import Session as vici_session
+
+    try:
+        session = vici_session()
+    except Exception:
+        raise ViciInitiateError("IPsec is not initialized")
```

Comment:
A minor note: I wonder if this can occur when IPsec is in fact configured, and whether we can or should try to differentiate that from the expected error when StrongSWAN simply isn't running.

Reply:
Would you like me to check if IPSEC is configured? If not, generate ''IPsec is not configured' else 'IPSec is not running'.